Index.html

```diff
@@ -7,6 +7,7 @@
     <title>Portfolio</title>
     <!-- Css fie  -->
     <link rel="stylesheet" href="Css/master.css" />
+    <link rel="stylesheet" href="Css/Deskto.css" media="(min-width: 768px) ">
```

Comment:
- [ ] Great job setting the media query from here. Please remember the guidelines are to test and prepare you on some particular aspect, and on this you were to use media queries for the Desktop breakpoint in the CSS file. This was to see how you can differentiate the mobile and desktop in one CSS file. Please, you can use the other method later on. It was clearly mentioned in the project requirement for you as well 👇🏾 


![Screenshot (917)_LI](https://user-images.githubusercontent.com/65298048/228689872-fac3f1d6-23de-45d5-b1f5-431e5e661629.jpg)